store/sqlstore/channel_store.go

```diff
@@ -1694,9 +1694,8 @@ func (s SqlChannelStore) RemoveMember(channelId string, userId string) store.Sto
 	})
 }
 
-func (s SqlChannelStore) RemoveAllDeactivatedMembers(channelId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		query := `
+func (s SqlChannelStore) RemoveAllDeactivatedMembers(channelId string) *model.AppError {
+	query := `
 			DELETE
```

Comment:
Do you mind to unindent the query string too?

Reply:
Done.